instrumentation/afl-gcc-pass.so.cc

```diff
@@ -714,9 +714,11 @@ struct afl_pass : gimple_opt_pass {
 
   }
 
+  /* Returns the source file name attached to the function declaration F. If
+     there is no source location information, returns an empty string.  */
   std::string getSourceName(function *F) {
 
-    return DECL_SOURCE_FILE(F->decl);
+    return DECL_SOURCE_FILE(F->decl) ? "" : DECL_SOURCE_FILE(F->decl);
```

Comment:
I didn't look into it deeply but isn't this flipped? If there is a file, it returns empty string?

Reply:
You are correct, let me fix that and check why this has not been spotted before!